build.gradle

```diff
@@ -90,7 +93,28 @@ githubPages {
 }
 
 shadowJar {
-    classifier = 'all'
+    baseName = 'launchdarkly-client'
+    //no classifier means that the fat + shaded jar becomes the default artifact
+    classifier = ''
+
+    // Shade all jars except for launchdarkly
+    relocate('com', 'com.launchdarkly.shaded.com') {
```

Comment:
It might be appropriate to reconsider this approach. In my project, SBT (sbt-assembly) got confused by `launchdarkly-client` depending on `com.google.code.gson:gson`, but both artifacts included `com/google/gson/Gson.class` etc.